### PR TITLE
PMM-10457 Moved docs from pmm repo

### PR DESCRIPTION
**Pull request by gen1us2k**
PMM-0

Build: SUBMODULES-0

- [ ] Links to other linked pull requests (optional).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+dbaas-controller's Kubernetes client is fully configured with kubeconfig file. It contains information about Kubernetes API server address, authentication and authorization mechanisms, default namespace, etc. The whole kubeconfig file content is passed to the client constructor.
+
+#### Kubernetes client revised
+
+When it made a lot of sense to start with kubectl CLI, number of months later (Apr 2022) it is not anymore as it means we probably need to update it quite often and maybe support number of those binaries in our code. Also it is against k8s tools skew version policies to maintain old binaries in our code.
+
+Also client-go library advanced, it is support number of auth and also quite good compatibility matrix:
```
Comment (percona-csalguero) [inline CONTRIBUTING.md:34]:
```suggestion
Also client-go advanced, it now supports several authentication mechanisms and also a quite good compatibility matrix:
```

File: README.md [hunk 2]
```diff
@@ -4,4 +4,35 @@
...
+
+We encourage contributions and are always looking for new members that are as dedicated to serving the community as we are.
+
+If you’re looking for information about how you can contribute, we have [contribution guidelines](CONTRIBUTING.md) across all our repositories in `CONTRIBUTING.md` files. Some of them may just link to the main project’s repository’s contribution guidelines.
+
+We're looking forward to your contributions and hope to hear from you soon on our [Forums](https://forums.percona.com) and [Discord](https://per.co.na/discord).
+
+## Submitting Bug Reports
+
+If you find a bug in Percona Monitoring and Management  or one of the related projects, you should submit a report to that project's [JIRA](https://jira.percona.com) issue tracker. Some of related project also have GitHub Issues enabled, so you also could submit there.
```
Comment (percona-csalguero) [inline README.md:20]:
```suggestion
If you find a bug in Percona Monitoring and Management  or one of the related projects, you can submit a report to that project's [JIRA](https://jira.percona.com) issue tracker. Some of the related projects also have GitHub Issues enabled, so you can also submit there.
```

**Review by percona-csalguero — APPROVED** [review]
I made some minor minor changes.
I would also request the review of our tech writers.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+### The Rules
+
+1. Start Feature Build for every feature/improvement you are working on.
+2. Start PullRequest to percona-lab/pmm-submodules as DRAFT.
+3. Change the status of Pull Request from Draft to Open ONLY if your changes must be merged to pmm-submodules.
+4. Include a short explanation in the Long Description field of the Feature in PR for feature build and checkboxes to all related Pull Requests. Check other [PRs](https://github.com/Percona-Lab/pmm-submodules/pulls) as examples.
+5. After all related PRs in feature build are merged you should:
+   a. either close the PR and delete the branch (this is the default option) or
+   b. merge the PR to pmm-submodules repository (please note, this rarely needs to be merged, for example infrastructure changes do)
```
Comment (BupycHuk) [inline CONTRIBUTING.md:147]: do we need to duplicate these parts? isn't it better to just point them to pmm docs? Having them in both places makes it harder to manage.

**Review by BupycHuk — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (gen1us2k) [reply CONTRIBUTING.md:147]: I think that we'll move to GitHub for the developer documentation. I agree that it makes harder to manage but it's more friendly for newbies.

Comment (BupycHuk) [reply CONTRIBUTING.md:147]: I didn't get you. PMM docs are in Github as well, so no links outside of GitHub.

**Pull request merged**